backend/custom_plugins/models.py

```diff
@@ -14,6 +14,15 @@ def __str__(self):
         return self.name
 
 class Blurb(CMSPlugin):
+    TYPE_CHOICES = (
+        ('link', 'Link'),
```

Comment:
Please internationalize the human-readable values, e.g. `Link` and `Button`. Read more about Django internationalization here: https://docs.djangoproject.com/en/2.1/topics/i18n/translation/

Hint: It should be `('link', _('Link'))`.